### ISSUE TITLE
make storageIdentifier field searchable (add to Solr)

### ISSUE THREAD

**Issue by kamil386** [IQSS/dataverse#6911]
How to add storageIdentifier field into solr?

**Comment by pdurbin** [comment]
@kamil386 hi! Here are some screenshots that I hope help you get started.

<img width="1380" alt="Screen Shot 2020-05-13 at 10 52 10 AM" src="https://user-images.githubusercontent.com/21006/81828574-eeaf5880-9507-11ea-9585-c9ef22b9c60f.png">
<img width="1380" alt="Screen Shot 2020-05-13 at 10 52 18 AM" src="https://user-images.githubusercontent.com/21006/81828580-f0791c00-9507-11ea-9b4d-a2fc4b8532c6.png">
<img width="1380" alt="Screen Shot 2020-05-13 at 10 52 33 AM" src="https://user-images.githubusercontent.com/21006/81828585-f242df80-9507-11ea-82d7-101405de793d.png">
<img width="1380" alt="Screen Shot 2020-05-13 at 10 52 55 AM" src="https://user-images.githubusercontent.com/21006/81828589-f2db7600-9507-11ea-8956-eb19e12c8157.png">

In short, I always start with `SearchFields.java`. I'd suggest looking at "fileContentType" as a model. In addition to changing the Java code, you'll need to update schema.xml.

I'm wondering if there are any security concerns about putting this information in Solr but I suppose not. I'd be curious to hear about your use case.

**Comment by pdurbin** [comment]
@kamil386 are you still interested in this? Did the screenshots help?
